ReactWindows/ReactNative.Shared/Modules/Core/ReactChoreographer.cs

```diff
@@ -82,7 +116,7 @@ private ReactChoreographer(CoreApplicationView applicationView)
         public event EventHandler<FrameEventArgs> IdleCallback;
 
         /// <summary>
-        /// The choreographer instance.
+        /// The choreographer main instance.
         /// </summary>
         public static IReactChoreographer Instance
```

Comment:
Do we need this special case? Is it more developer-friendly to have this "main" `Instance` and an explicit alt `ThreadInstance`, or should all calls to `Instance` be specific to the thread?

Reply:
I think I know where you're headed, but ReactChoreographer.Instance is called from random threads (module one, etc.), making any thread affinity dangerous.

---
In reply to: [225740827](https://github.com/Microsoft/react-native-windows/pull/2000#discussion_r225740827) [](ancestors = 225740827)